index.html

```diff
@@ -0,0 +1,199 @@
+                </ul>
+                <a href="#" class="button"><img src="images/see-project.png" alt="Seeproject"></a>
+            </div>
+        </section>
+    </div>
+    <div class="card">
+        <section class="profile-picture">
+            <img src="images/snapshort-profile-2.png" alt="Snapshot-profile">
+        </section>
+        <section class="tonic">
+            <h1>Multi-Post<br>Stories</h1>
+            <div class="list">
+                <ul class="list">
+                    <li><p>CANOPY</p></li>
+                    <li><img src="images/counter-1.png" alt="counter1"></li>
+                    <li><p class="stretch">Back End Dev</p></li>
+                    <li><img src="images/counter-2.png" alt="counter2"></li>
+                    <li><p>2015</p></li>
+                </ul>
+            </div>
+            <p class="contents">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <br>
+            <div class="buttons">
+                <ul class="list">
+                    <li><a href="#" class="button"><img src="images/html.png" alt=".html"></a></li>
+                    <li><a href="#" class="button"><img src="images/css.png" alt=".css"></a></li>
+                    <li><a href="#" class="button"><img src="images/javascript.png" alt=".js"></a></li>
+                </ul>
+                <a href="#" class="button"><img src="images/see-project.png" alt="Seeproject"></a>
+            </div>
+        </section>
+    </div>
+    <div class="card">
+        <section class="profile-picture">
+            <img src="images/snapshort-profile-3.png" alt="Snapshot-profile">
+        </section>
+        <section class="tonic">
+            <h1>Tonic</h1>
+            <div class="list">
+                <ul class="list">
+                    <li><p>CANOPY</p></li>
+                    <li><img src="images/counter-1.png" alt="counter1"></li>
+                    <li><p>Back End Dev</p></li>
+                    <li><img src="images/counter-2.png" alt="counter2"></li>
+                    <li><p>2015</p></li>
+                </ul>
+            </div>
+            <p class="contents">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <br>
+            <div class="buttons">
+                <ul class="list">
+                    <li><a href="#" class="button"><img src="images/html.png" alt=".html"></a></li>
+                    <li><a href="#" class="button"><img src="images/css.png" alt=".css"></a></li>
+                    <li><a href="#" class="button"><img src="images/javascript.png" alt=".js"></a></li>
+                </ul>
+                <a href="#" class="button"><img src="images/see-project.png" alt="Seeproject"></a>
+            </div>
+        </section>
+    </div>
+    <div class="card">
+        <section class="profile-picture">
+            <img src="images/snapshort-profile-4.png" alt="Snapshot-profile">
```

Comment:
- [ ] For every project image, there image is not responsive for all mobile screen sizes. Give this image flexible dimensions that will allow it to fit perfectly on all screen sizes. You may use (% or rem)
![Screenshot from 2021-08-14 19-12-30](https://user-images.githubusercontent.com/65501535/129452632-f9647d85-7513-4a4f-a318-112fee4d8eb2.png)